packages/blade/src/components/BottomSheet/index.ts

```diff
@@ -0,0 +1,8 @@
+export {
+  BottomSheet as PrereleaseBottomSheet,
```

Comment:
I want it to SCREAM this is unstable 😆 
```suggestion
  BottomSheet as BottomSheet_PRE_RELEASE,
```

Reply:
lol sure.